apps/web/app/[locale]/timesheet/[memberId]/components/CalendarView.tsx

```
@@ -228,35 +228,36 @@ const BaseCalendarDataView = ({ data, daysLabels, t, CalendarComponent }: BaseCa
                                                     )}
-                                                    showSize={true}
+                                                    showSize={false}
                                                     dash
                                                     taskNumberClassName="text-sm"
                                                 />
-                                                <div className="flex items-center gap-x-2">
-                                                    {task.project && <ProjectLogo imageUrl={task.project.imageUrl as string} />}
-                                                    <span className="flex-1 font-medium">{task.project && task.project.name}</span>
+                                                <div className="flex flex-row items-center  py-0 gap-2  flex-none order-2 self-stretch flex-grow-0">
+                                                    {task.project && <ProjectLogo className="w-[28px] h-[28px] drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] rounded-[8px]" imageUrl={task.project.imageUrl as string} />}
+                                                    <span className="!text-ellipsis !overflow-hidden !truncate !text-[#3D5A80] dark:!text-[#7aa2d8]">{task.project && task.project.name}</span>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)